gtsfm/loader/hilti_loader.py

```diff
@@ -41,13 +42,14 @@
     4: "calib_3_cam4-camchain-imucam.yaml",
 }
 
+GT_POSE_RELATIVE_PATH = "lidar/fastlio2_odom.txt"
 LIDAR_POSE_RELATIVE_PATH = "lidar/fastlio2.g2o"
 LIDAR_CONSTRAINTS_RELATIVE_PATH = "lidar/constraints.txt"
 IMAGES_FOLDER = "images"
 
-HARD_RELATIVE_POSE_PRIOR_SIGMA = np.ones((6,)) * 1e-3  # CAM_IMU_POSE_PRIOR_SIGMA in BA should have similar value
-SOFT_RELATIVE_POSE_PRIOR_SIGMA = np.ones((6,)) * 3e-2
-SOFT_ABSOLUTE_POSE_PRIOR_SIGMA = np.ones((6,)) * 3e-2
+HARD_POSE_PRIOR_COVARIANCE = np.eye(6) * 1e-8
+STATIONARY_POSE_PRIOR_COVARIANCE = np.eye(6) * 1e-7
+CAM2_BACKBONE_POSE_PRIOR_COVARIANCE = np.diag(np.array([np.deg2rad(30.0), np.deg2rad(30.0), np.deg2rad(30.0), 1, 1, 1]))
```

Comment:
1 meter? is this for relative pose prior?

Reply:
Yes. This is what we used for the experiments. I think we should have made it a bit tighter?

Reply:
Yeah, I think it should be tighter. There are from Bayesian ICP, not fastlio right? 

But it depends on the ratio of priors to two-view camera edges we have, and the prior of the camera edge itself. Ultimately, we should tune to maximize performance.

Reply:
I think lets leave it for now, given that we would have to redo the experiments with a tighter prior.

Reply:
Add a TODO comment.